inc/Engine/Preload/Controller/PreloadUrl.php

```diff
@@ -154,12 +154,18 @@ public function preload_url( string $url ) {
 				$duration = ( microtime( true ) - $start ); // Duration of the request.
 			}
 
+			$default_delay_between = 500000;
+
 			/**
 			 * Filter the delay between each preload request.
 			 *
-			 * @param float $delay_between the defined delay.
+			 * @param int $delay_between the defined delay.
 			 */
-			$delay_between = apply_filters( 'rocket_preload_delay_between_requests', 500000 );
+			$delay_between = apply_filters( 'rocket_preload_delay_between_requests', $default_delay_between );
```

Comment:
Yes I thought about this case and this can be like that:
```
$delay_between = (int) apply_filters( 'rocket_preload_delay_between_requests', $default_delay_between );
if ( empty( $delay_between ) ) {
	$delay_between = $default_delay_between;
}
```

for `hello` case it will be cast to be zero and that's empty so we will make it the default value.

I'm OK with both solutions, I was just thinking with u.